service/project.go

```diff
@@ -373,13 +372,19 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 				uis.LoggedError(w, r, http.StatusInternalServerError, errors.Wrapf(err, "error checking for commit queue aliases"))
 				return
 			}
-			if len(aliases) == 0 {
+
+			remainingAliases := 0
+			for _, a := range aliases {
```

Comment:
what if the user just added a commit queue alias now?

Reply:
We only run through this if `len(responseRef.CommitQueueAliases) == 0`, i.e. no new commit queue aliases.